src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/TableConcatenate.txt

```diff
@@ -0,0 +1,96 @@
+
+>> Table([{a:0, b:false, c:"Hello"}], [{a:1, b:true, c:"World"}])
```

Comment:
>>> Table([{a:0, b:fals [](http://example.com/codeflow?start=0&length=22)

Suggestion: rename this file to Table.txt (there's no TableConcatenate function)

Reply:
saw TableCoercion.txt so thought I could leave it as is